civictechindexadmin/data/models.py

```diff
@@ -81,6 +81,21 @@ def __str__(self):
         return f"Alias {self.id}: {self.tag} {self.alias}"
 
 
+class Contact(models.Model):
+    email = models.CharField(max_length=128, unique=True)
+    text = models.TextField(max_length=4096, null=True, blank=True)
```

Comment:
For TextField and CharFields, best practice is to ONLY use "blank=True" - that way the row will always have an empty string for missing data in that field. If you allow null, then you would have to check for null and '' to decide if you don't have a value.